examples/vault-agent/main.tf

```diff
@@ -13,7 +13,7 @@ terraform {
 resource "aws_instance" "example_auth_to_vault" {
   ami           = "${var.ami_id}"
```

Comment:
I'm confused, shouldn't the examples also be updated? So shouldn't this be 
`ami           = var.ami_id` now?

Reply:
Yes, I hadn't pushed those commits for some reason. 

bbe0579
7027e0a
bbdf145
0ba9b1a
68a022e
2217a13